data/changelog/ocaml/2024-05-13-ocaml-5.2.0.md

```diff
@@ -483,88 +484,88 @@ changelog: |
   - [#12819](https://github.com/ocaml/ocaml/issues/12819): Clarify which runtime interactions are allowed in custom ops
     (Basile Clément, review by Guillaume Munch-Maccagnoni and Xavier Leroy)
   
-  - [#12840](https://github.com/ocaml/ocaml/issues/12840): manual: update runtime tracing chapter for custom events (ex [#12335](https://github.com/ocaml/ocaml/issues/12335))
+  - [#12840](https://github.com/ocaml/ocaml/issues/12840): Manual: update runtime tracing chapter for custom events (e.g., [#12335](https://github.com/ocaml/ocaml/issues/12335))
     (Lucas Pluvinage, Sadiq Jaffer and Olivier Nicole, review by Gabriel Scherer,
-     David Allsopp, Tim McGilchrist and Thomas Leonard)
+     David Allsopp, Tim McGilchrist, and Thomas Leonard)
   
-  - [#13066](https://github.com/ocaml/ocaml/issues/13066), update OCAMLRUNPARAM documentation for the stack size parameter l
+  - [#13066](https://github.com/ocaml/ocaml/issues/13066): Update `OCAMLRUNPARAM` documentation for the stack size parameter l
```

Comment:
`l` ⇒ `` `l` ``: `l` is the name of the stack size parameter.

Reply:
Thank you! Changed!